.github/workflows/event-pull_request.yml

```diff
@@ -46,8 +46,9 @@ jobs:
       test-config: QUICK=1
       san: address
       env: ubuntu-latest
-      container: ubuntu:jammy
+      container: ubuntu:latest
```

Comment:
Is there a reason to use a container here? Why not running directly on the runner? We also won’t need the additional step to install git

Reply:
IIRC that GH runner comes with built in clang which is old, and we need to adjust the symlink. I want to build on clean image